DEVELOPMENT.md

```diff
@@ -0,0 +1,67 @@
+## Development
+
+### Code Style
+
+We recommend you use IntelliJ as your IDE. The code style template for the project can be found in the [codestyle](https://github.com/airlift/codestyle) repository along with our general programming and Java guidelines. In addition to those you should also adhere to the following:
+
+* Alphabetize sections in the documentation source files (both in the table of contents files and other regular documentation files). In general, alphabetize methods/variables/sections if such ordering already exists in the surrounding code.
```

Comment:
I there an ordering to this section list most important to least, or grouped by category?

Reply:
I didn't realize that most of this was moved from somewhere else. You don't need to deal with the organization.